lib/features/base/base_controller.dart

```diff
@@ -372,6 +374,23 @@ abstract class BaseController extends GetxController
     }
   }
 
+  void injectWebSocket(Session? session, AccountId? accountId) {
+    try {
+      requireCapability(
```

Comment:
beside capability, IMO, we should check the flag: `supportPush` before initialize web socket or not.